src/mui/input/CheckboxGroupInput.js

```diff
@@ -107,16 +139,25 @@ export class CheckboxGroupInput extends Component {
 
     render() {
         const { choices, label, resource, source } = this.props;
+        const styles = getStyles(this.props, this.context, this.state);
+
         return (
-            <Labeled label={label} source={source} resource={resource}>
-                <div>
-                    {choices.map(this.renderCheckbox)}
+            <div>
+                <div style={styles.labelContainer}>
+                    <div style={styles.label}>
+                        <FieldTitle label={label} source={source} resource={resource} />
+                    </div>
                 </div>
-            </Labeled>
+                {choices.map(this.renderCheckbox)}
+            </div>
         );
     }
 }
 
+CheckboxGroupInput.contextTypes = {
```

Comment:
I prefer to decorate with the `muiThemeable` HOC to have the theme as prop rather than context. Thiat will simplify your tests, too. See `mui/layout/AppBar.js` for an example.